layers/raw.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2012 Google, Inc. All rights reserved.
+// Copyright 2009-2011 Andreas Krennmair. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
+	"fmt"
+	"github.com/google/gopacket"
+)
+
+// RawIP (DLT_RAW) contains no header and we start with the IP header
+type Raw struct {
```

Comment:
This is great code, but I'm not 100% sure its specific purpose... why would this be useful instead of just having a packet with either `IPv4` or `IPv6` as the first layer?